site/content/usage/20-schema.md

```diff
@@ -504,6 +532,34 @@ OwnerReference:
   - name
   - uid
   type: object
+Profile:
+  additionalProperties: false
+  properties:
+    revision:
+      type: string
+    source:
+      type: string
+  type: object
+Repo:
+  additionalProperties: false
+  properties:
+    branch:
+      type: string
+    email:
+      type: string
+    fluxSubDir:
```

Comment:
Rename this one as well to `fluxDir`, I though this is was generated...

Reply:
It is supposed to be generated, yes, but this doesn't seem to consistently re-generate things on my machine, not sure why/what I'm doing wrong! 🤔

Reply:
OK, it seems to generate it under `github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5/zz_generated.deepcopy.go` instead of `pkg/apis/eksctl.io/v1alpha5/zz_generated.deepcopy.go` for some reason. I'll try to debug this and fix it in another PR.